codes/quantum/oscillators/cat_code.yml

```diff
@@ -0,0 +1,97 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#},
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are 
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: cat_code
+physical: oscillators
+logical: qubits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'Cat code'
+
+# First reference with the code (optional).
+introduced: '\(|\alpha\rangle \pm |-\alpha\rangle\) cat code \cite{doi:10.1103/PhysRevA.59.2631}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'It is a continuous-variable code that encode a logical qubit into a harmonic oscillator. In phase space, cat code has rotation symmtry which can protect against dephasing noise. '
```

Comment:
Please define using coherent states. For example, something like the description here: https://arxiv.org/abs/1503.00194. Introduce d = S+1 and use instead of S to match notion of "distance". Check out bosonic_rotation and bosonic_binomial entries by jtiosue and fock_state entry by me which explains this feature more generally.